src/Helpers/TypesEnum.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+enum TypesEnum: string
+{
+    case MODEL = "model";
```

Comment:
Please use PascalCase for enumeration types: https://www.php.net/manual/en/language.enumerations.basics.php

```suggestion
    case Model = "model";
```